examples/models/llama2/runner/runner.cpp

```diff
@@ -248,12 +248,12 @@ Error Runner::generate(
       "Sequence length exceeded - please increase the seq_len value passed to generate()");
 
   // start the main loop
-  int64_t pos = 0; // position in the sequence
+  int32_t pos = 0; // position in the sequence
```

Comment:
Any specific reason we cast from `int64_t` to `int32_t`?

Reply:
Because we don't support int64 well in Qnn HTP such as the index tensor of ScatterND op.